templates/artist/includes/artist_detail_past.html

```diff
@@ -4,9 +4,58 @@
     <h2>PAST CAMPAIGNS</h2>
     <ul>
         {% for past_campaign in artist.past_campaigns %}
+        <div class="break">
+            <hr>
+        </div>
+        <div class="past-campaign">
             <li>
-                ${{ past_campaign.amount|intcomma }} {{ past_campaign.project.reason }}: {{ past_campaign.percentage_funded }}% funded [ended {{ past_campaign.end_datetime|timesince }} ago]
+                <h3>${{ past_campaign.amount|intcomma }} {{ past_campaign.project.reason }}: {{ past_campaign.percentage_funded }}% funded</h3>
+                <h4>[ended {{ past_campaign.end_datetime|timesince }} ago]</h4>
+                <!-- Countdown Widget -->
+                <div class="artist-detail-padding">
```

Comment:
Agree with the countdown widget. Not entirely sure what you are talking about with:
"pull in code that's duplicated in artist_detail_over.html and artist_detail_past.html from an artist/includes/ template."